pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/filter/postfilter/osteo/OldOsteoPostFilter.java

```diff
@@ -16,9 +18,12 @@ public class OldOsteoPostFilter extends BaseStudyPostFilter {
 
     private final StudyPostFilterStrategy<Activities> oldOsteoFilter;
 
+    private final NewOsteoPostFilterStrategy oldOsteoOncHistoryFilter;
+
     protected OldOsteoPostFilter(ElasticSearchParticipantDto elasticSearchParticipantDto, DDPInstanceDto ddpInstanceDto) {
         super(elasticSearchParticipantDto, ddpInstanceDto);
         this.oldOsteoFilter = new OldOsteoPostFilterStrategy();
+        this.oldOsteoOncHistoryFilter = new NewOsteoPostFilterStrategy(ddpInstanceDto);
```

Comment:
Not sure why 'New...' gets assigned to 'old...' here. If you have the context it would be great to leave a new class comment that includes a breif explanation of the difference between old and new filters, if that is in fact what old and new refer to.
And then maybe this will make more sense? If not maybe an explanation here as well?

Reply:
It is this code that led to the discussion we had and are having about the ticket (Whitney, Pegah, and I in Slack).  I've left it as-is for now with the intention of getting clarification and then coming back to it - I'm glad you saw that and it evoked the question.

If the conversation continues in the same direction, I think "NewOsteoPostFilterStrategy" will be renamed to something like "DDPInstanceIdFilterStrategy". That new strategy will be used by both the "New" (osteo2) and "Old" osteo filters.  I'll likely be adding additional object manipulation and documenting the updates in a class comment.

Reply:
@denniscunningham The PR is updated.  Let me know what you think about the changes.  If they look good, we can resolve this thread.